devel/allegro5/Portfile

```diff
@@ -13,8 +13,8 @@ subport ${name}-devel {}
 
 if {${name} eq ${subport}} {
     PortGroup       github 1.0
-    github.setup    liballeg allegro5 5.2.2.0
-    revision        1
+    github.setup    liballeg allegro5 5.2.4.0
+    revision        0
```

Comment:
Normally the `revision` line is simply deleted in this case.

Reply:
Updated